api/envoy/api/v2/core/health_check.proto

```diff
@@ -288,6 +289,11 @@ message HealthCheck {
   // If empty, no event log will be written.
   string event_log_path = 17;
 
+  // [#not-implemented-hide:]
+  // The gRPC service for the health check event service.
+  // If empty, health check events won't be sent to a remote endpoint.
+  GrpcService grpc_service = 22;
```

Comment:
For future proofing, I would add a `EventServiceConfig` message to the API and use it everywhere that `GrpcService` appears right now. Then put a `GrpcService` in there. In the future, this will give you the ability to add aggregated streaming without breaking any of the config sites.

Reply:
+1 good idea.

Reply:
done